glib/src/gstring.rs

```diff
@@ -88,6 +86,12 @@ impl Clone for GString {
     }
 }
 
+impl fmt::Debug for GString {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{:?}", self.as_str())
```

Comment:
```suggestion
       <&str as fmt::Debug>::fmt(self.as_str(), f)
```

That way you pass through all formatting options too.

Reply:
That's what I had first, but I thought it would be less readable 😄